schemas/com.snowplowanalytics.snowplow/pii_enrichment/jsonschema/1-0-0

```diff
@@ -0,0 +1,654 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a PII enrichment original values",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "pii_enrichment",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "pii": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "pojo": {
+            "type": "object",
+            "properties": {
+              "user_id": {
```

Comment:
Am I right that we're using snake_case here to match `EnrichedEvent`'s original format? If this is the only reason than I'm not sure there's a value in that:

1. We recently decided to standardize on camelCase
2. We often transform key's cases between their original format and schema-based
3. I believe `EnrichedEvent` should and will use camelCase someday

However, even with that - this is not a strong opinion, I generally have nothing against snake_case.

Reply:
Moving to generic pojoField